### PR TITLE
vNext Asset Clean-up

### PR DESCRIPTION
**Pull request by keveleigh**
Overview
---
Reverted some GUIDs to their value in master to ease porting.

Refactored out a SceneDescriptionPanel prefab for ease of updating theme across all example scenes.

Turned off the standalone resolution dialog.

Updated an incorrectly copied comment in MixedRealityConfigurationProfile.

Updated the MRTK Logo and added an additional light variant, to add support for both the Unity Pro theme and the Unity Personal theme.

### CODE REVIEW

File: Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Controllers/MixedRealityControllerVisualizer.cs.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 9746d77301599614e874ca00077b49b3
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Controllers/MixedRealityControllerVisualizer.cs.meta]: Why did this meta file change?

File: Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Utilities/InteractibleHighlight.cs.meta [hunk 2]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: dd10302ee779a324bb76f623fd767559
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Utilities/InteractibleHighlight.cs.meta]: Why meta file change?

**Comment by StephenHodgson** [comment]
> Reverted some GUIDs to their value in master to ease porting.

I think that's a bad idea. Besides we told everyone that htk porting was not going to be possible.

**Comment by StephenHodgson** [comment]
> Refactored out a SceneDescriptionPanel prefab for ease of updating theme across all example scenes.

While we're at it, let's update it to use Text Mesh Pro.

**Comment by keveleigh** [comment]
>I think that's a bad idea. Besides we told everyone that htk porting was not going to be possible.

We can tell people it's not easy, but files that are basically the same shouldn't disappear. Most of them are already checked in identically to master, so should we regenerate literally every meta instead?

>While we're at it, let's update it to use Text Mesh Pro.

See https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2513#discussion_r206291214

**Comment by StephenHodgson** [comment]
> Most of them are already checked in identically to master
That's probably not true.  I've been intentionally regenerating the meta files as I've been porting it over.

**Comment by StephenHodgson** [comment]
What's the difference between Material Gallery and Standard Materials? Can we get rid of one of these scenes?

**Comment by SimonDarksideJ** [comment]
Agreed, we only need one scene demonstrating MRTK features.
I believe the standard scene is to show the comparison, but that's not needed in MRTK.  maybe just keep screenshots of it for documentation, or in a private github repo we can link to for reference.

**Comment by keveleigh** [comment]
Standard Materials shows off the different MRTK_Standard_[color] materials that we provide, while Material Gallery shows off the different features of the MRTK Standard shader.

**Comment by SimonDarksideJ** [comment]
Daft Question @keveleigh but don't the materials already do that themselves?  in the inspector.
Just qualifying as it seems a but much to have a scene to show "basic" colors when I can see them already.

**Comment by keveleigh** [comment]
Yeah, probably. I have no opinion on its existence but am relaying the contents of it :)

image

**Comment by keveleigh** [comment]
@StephenHodgson Assuming he also updated the MaterialGallery scene, likely.

**Comment by keveleigh** [comment]
>That's probably not true. I've been intentionally regenerating the meta files as I've been porting it over.

@StephenHodgson 

Of the 99 script/prefab/material/etc (non-folder, non-readme) meta files that share a name between master and vNext, 81 of them share a GUID, while 18 have different GUIDs. This does not include files that are basically the same but have been renamed, like the solvers (which all have the same GUIDs as master too). After this PR, those numbers change to 86 same and 12 different.

> SpeechInputSource.cs | same
> TouchscreenInputSource.cs | same
> DebugInteractionSourcePose.cs | same
> DebugInteractionSourceState.cs | same
> USBDeviceInfo.cs | same
> USBDeviceListener.cs | same
> BuildDeployWindow.cs | same
> BuildInfo.cs | same
> CopyDirectoryInfo.cs | same
> InputMappingAxisUtility.cs | diff
> UwpProjectPostProcess.cs | same
> CursorStateEnum.cs | same
> FocusDetails.cs | same
> RayStep.cs | same
> AnimatorParameter.cs | same
> Edge.cs | diff
> Handedness.cs | diff
> InscribedRectangle.cs | diff
> BaseInputEventData.cs | same
> DictationEventData.cs | same
> InputEventData.cs | same
> SourceStateEventData.cs | same
> SpeechEventData.cs | same
> BoundsExtensions.cs | same
> CameraExtensions.cs | same
> Color32Extensions.cs | same
> ComponentExtensions.cs | same
> EnumerableExtensions.cs | same
> EventSystemExtensions.cs | same
> GameObjectExtensions.cs | same
> InteractionSourceExtensions.cs | same
> LayerExtensions.cs | same
> MathfExtensions.cs | same
> ReflectionExtensions.cs | same
> TransformExtensions.cs | same
> VectorExtensions.cs | same
> CanvasEditorExtension.cs | diff
> ICursorModifier.cs | diff
> selawk.ttf | same
> selawkb.ttf | same
> selawkl.ttf | same
> selawksb.ttf | same
> selawksl.ttf | same
> 3DTextSelawik.mat | diff
> 3DTextSelawikBold.mat | diff
> 3DTextSelawikLight.mat | diff
> 3DTextSelawikSemibold.mat | diff
> 3DTextSelawikSemilight.mat | diff
> MRTK_Logo.png | diff
> EnforceEditorSettings.cs | same
> EditorPrefsUtility.cs | same
> Solver.cs | same
> SolverHandler.cs | same
> BaseRayStabilizer.cs | same
> GazeStabilizer.cs | same
> InterpolationUtilities.cs | same
> Interpolator.cs | same
> RaycastResultComparer.cs | same
> TwoHandMoveLogic.cs | diff
> TwoHandRotateLogic.cs | diff
> TwoHandScaleLogic.cs | diff
> VectorRollingStatistics.cs | same
> Response.cs | diff
> AdapterInfo.cs | same
> DevicePortalConnections.cs | same
> DHCPInfo.cs | same
> InstallStatus.cs | same
> IpAddressInfo.cs | same
> MachineName.cs | same
> ProcessList.cs | same
> CalibrationSpace.cs | same
> CameraCache.cs | same
> ClipPlane.cs | same
> WindowsApiChecker.cs | same
> SpeechInputHandler.cs | same
> DefaultCursor.prefab | diff
> AnimatedCursor.cs | same
> CursorModifier.cs | same
> InteractiveMeshCursor.cs | same
> MeshCursor.cs | same
> ObjectCursor.cs | same
> SpriteCursor.cs | same
> BaseControllerPointer.cs | diff
> Billboard.cs | same
> HoverLight.cs | same
> MRTK_Standard_Blue.mat | same
> MRTK_Standard_Charcoal.mat | same
> MRTK_Standard_Cyan.mat | same
> MRTK_Standard_Emerald.mat | same
> MRTK_Standard_Gray.mat | same
> MRTK_Standard_Green.mat | same
> MRTK_Standard_Lime.mat | same
> MRTK_Standard_Magenta.mat | same
> MRTK_Standard_Orange.mat | same
> MRTK_Standard_Pink.mat | same
> MRTK_Standard_Purple.mat | same
> MRTK_Standard_Red.mat | same
> MRTK_Standard_White.mat | same
> MRTK_Standard_Yellow.mat | same
> Same: 81 | Diff: 18

**Comment by StephenHodgson** [comment]
huh, interesting cause I had not been copying the meta files when porting features.

**Comment by davidkline-ms** [comment]
>Daft Question @keveleigh but don't the materials already do that themselves? in the inspector.
Just qualifying as it seems a but much to have a scene to show "basic" colors when I can see them already.
Might be better to just have a document page the user can view without having to navigate a scene.

Fair enough. This scene was mine. I am perfectly happy to have a doc page to show the colors, though it is often VERY handy to have them in device, particularly for HoloLens and other AR systems, to see what they *really* look like

**Comment by StephenHodgson** [comment]
Let's hold off merging this PR until the Material PR gets approved to make sure we don't have conflicts.

File: ProjectSettings/ProjectSettings.asset [hunk 3]
```diff
@@ -54,7 +54,7 @@ PlayerSettings:
   androidShowActivityIndicatorOnLoading: -1
   tizenShowActivityIndicatorOnLoading: -1
   iosAppInBackgroundBehavior: 0
-  displayResolutionDialog: 1
+  displayResolutionDialog: 2
```
Comment (StephenHodgson) [inline ProjectSettings/ProjectSettings.asset]: What does this do?

File: Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity [hunk 4]
```diff
@@ -1213,6 +1213,7 @@ Transform:
   m_Children:
   - {fileID: 1420623844}
   - {fileID: 1491536846}
+  - {fileID: 1237288067}
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity]: I take it that this removes the TMPro Description panel I put in?

Comment (keveleigh) [reply Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity]: It does not! It updates the MRTK logo with the updated version I added (which is transparent, so I also removed the transparent logo you added that was identical to this new one).

**Comment by StephenHodgson** [comment]
One thing we might want to do as well is add the default cursor to the focus provider

File: Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity (hunk 4 above)
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity]: What did you think about the text resolution with the SDF?

**Comment by StephenHodgson** [comment]
Also, could we change the project settings for Single Pass Instanced back to Multi Pass?  I find the perf really bad in Unity WMR. It stutters every 2-3 seconds.

File: ProjectSettings/ProjectSettings.asset (hunk 3 above)
Comment (keveleigh) [reply ProjectSettings/ProjectSettings.asset]: Changes "[Display Resolution Dialog](https://docs.unity3d.com/Manual/class-PlayerSettingsStandalone.html)" from "Enabled" to "Hidden By Default".

image

This is the popup that shows up after you build for Standalone and launch the exe. With this change, that popup only appears if you launch the app while holding alt. I've found that it helps for ease of launching after building, especially when a headset is involved.

**Comment by keveleigh** [comment]
>One thing we might want to do as well is add the default cursor to the focus provider

Is that a prefab or something I should update in each scene?

**Comment by StephenHodgson** [comment]
Select the "Head" object and open the gaze provider.
It'll be the first field

**Comment by StephenHodgson** [comment]
> but can’t “fly around.”

I don't think we have those kinds of interactions set up yet.

**Comment by keveleigh** [comment]
>change the project settings for Single Pass Instanced back to Multi Pass? I find the perf really bad in Unity WMR. It stutters every 2-3 seconds.

I'm not able to repro the stuttering, but I've reverted it.

**Comment by StephenHodgson** [comment]
Boundary Manager description panel should be updated to include OpenVR Headsets.

**Comment by Cameron-Micka** [comment]
@StephenHodgson thank you for the info! That explains why I couldn’t find anything when poring over the components. Haha

**Comment by davidkline-ms** [comment]
> Boundary Manager description panel should be updated to include OpenVR Headsets.

Isnt it sufficient to say "Immersive Headsets"? Otherwise we will eventually have to add *all* implemented VR systems, wont we?

**Comment by keveleigh** [comment]
>Boundary Manager description panel should be updated to include OpenVR Headsets.

>Isnt it sufficient to say "Immersive Headsets"? Otherwise we will eventually have to add all implemented VR systems, wont we?

Is there action to be taken here? I'm good either way.

**Comment by davidkline-ms** [comment]
> Boundary Manager description panel should be updated to include OpenVR Headsets.

> Isnt it sufficient to say "Immersive Headsets"? Otherwise we will eventually have to add all implemented VR systems, wont we?

> Is there action to be taken here? I'm good either way.

I would say leave as is. I am updating the scene in my boundary sdk pr and have panel updates to make

**Comment by keveleigh** [comment]
>I would say leave as is. I am updating the scene in my boundary sdk pr and have panel updates to make

Great! I think everything else has been addressed, but feel free to ping something if not.

**Pull request merged**